scout/server/blueprints/panels/views.py

```diff
@@ -254,14 +253,17 @@ def gene_edit(panel_id, hgnc_id):
             info_data["custom_inheritance_models"] = info_data["custom_inheritance_models"].split(
                 ","
             )
-
         store.add_pending(panel_obj, hgnc_gene, action=action, info=info_data)
         return redirect(url_for(".panel", panel_id=panel_id))
 
     if panel_gene:
-        form.custom_inheritance_models.data = ", ".join(
-            panel_gene.get("custom_inheritance_models", [])
-        )
+        custom_models = [
+            model for model in panel_gene.get("custom_inheritance_models", []) if model != ""
+        ]
+        if custom_models:
```

Comment:
Nice! Just out of curiosity, why not .join(custom_models) next?

Reply:
In case panel_gene["custom_inheritance_models"] contains elements that are ""

Reply:
Fixed in the parsing now. But I'd keep this code here, just in case loaded panels contain "" as well!